store.ts

```diff
@@ -15,6 +15,12 @@ export default class Store {
   };
   public delete = (key: string) => {
     this.data.delete(key);
+
+    const subscriptions = this.subscriptions.get(key);
+
+    if (subscriptions) {
+      subscriptions.forEach(fn => fn(null));
```

Comment:
`undefined` ?

Reply:
You're probably right because when you're calling the `get` of a `Map` with a non-existing key you'll get `undefined`